code/game/objects/items/storage/book.dm

```diff
@@ -122,6 +122,15 @@ GLOBAL_LIST_INIT(bibleitemstates, list("bible", "koran", "scrapbook", "burning",
 	if(!ishuman(L))
 		return
 	var/mob/living/carbon/human/H = L
+
+	// BEGIN ORBSTATION EDIT
+	if(HAS_TRAIT(H, TRAIT_DAMNED))
```

Comment:
This looks like it could be made modular, the only thing you gain by not overriding the proc somewhere else is not repeating the `ishuman` proc.

_Also_ it might not even be correct as implemented, any sect changes to bible behaviour would trigger before this, which seems wrong? You probably want this check to happen before anything else can.